### ISSUE TITLE
Can't click on `.prose-menu`

### ISSUE THREAD

**Issue by robsonsobral** [prose/prose#1077]
The stacking and `z-index` of home page elements makes impossible to click the `.prose-menu`.

**Comment by dereklieu** [comment]
Yeah this is an issue with breakpoints, the menu is visible at wider display sizes (above 1024 maybe?) but I agree at smaller sizes it's not ideal.
